lib/src/hydrated_bloc_storage.dart

```diff
@@ -26,31 +27,36 @@ class HydratedBlocStorage implements HydratedStorage {
   final Map<String, dynamic> _storage;
   final File _file;
 
+  /// `Lock` is used to synchronize access to IO
+  static final Lock lock = Lock();
```

Comment:
can we make this lock private?

Reply:
In next PR we did

Reply:
It was need for testing, before making storage non-singleton